src/providers/asgardeo.ts

```diff
@@ -0,0 +1,26 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export default function Atlassian<P extends {}>(
+  options: OAuthUserConfig<P> & {
+    domain: string
+    tenantId: string
+    scopes?: string
+  }
+): OAuthConfig<P> {
+  return {
+    id: "asgardeo",
+    name: "Asgardeo",
+    clientId: options?.clientId,
+    clientSecret: options?.clientSecret,
+    type: "oauth",
+    wellKnown: `https://${options.domain}/t/${options?.tenantId}/oauth2/token/.well-known/openid-configuration`,
+    authorization: {
+      params: { scope: options?.scopes || "openid email profile internal_login" } 
```

Comment:
Do we need `internal_login`?

Reply:
Yes we can omit that by using userinfo route. Updated the PR